packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -348,7 +348,7 @@ class MarkdownBuilder implements md.NodeVisitor {
       child = _buildRichText(
         TextSpan(
           style: _isInBlockquote
-              ? styleSheet.blockquote!.merge(_inlines.last.style)
+              ? _inlines.last.style!.merge(styleSheet.blockquote)
```

Comment:
Why is the force-unwrap safe here? I don't see any checks nearby that would validate that this is non-null.